modules/appeals_api/spec/factories/supplemental_claims.rb

```diff
@@ -25,6 +25,60 @@
     end
   end
 
+  factory :supplemental_claim_alt_signer, class: 'AppealsApi::SupplementalClaim' do
```

Comment:
See my other comment about modifying the headers just before the generator is run. We shouldn't need these factories since we don't really have a use for them outside of a single test each. Factories are added when they're expected to be used across multiple specs & spec files.